ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -91,6 +91,7 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
             ],
           },
         },
+        force_refresh: true,
```

Comment:
As an FYI, I'm going to be changing how we retrieve these sessions as it's probably worth just making one call to retrieve all sessions (and even load them into the store) rather than making separate calls for each target. I need to do this regardless for the active session filter

Reply:
good call, that would simplify this.